### ISSUE TITLE
Put See name in separate column

### ISSUE THREAD

**Issue by dfurber** [historyforge/historyforge#315]
On the people page, put the "see" name in a second column. Make it only appear if there is a "see name" in the list. Then make the name column less wide.

**Comment by PatientRowan** [comment]
I see your sneaky ways, you have made the View button extinct and made the Name clickable.  Will the variant name in the new column also be clickable?  That might be a more exact search than is advisable, as it eliminates near-hits; but it might be nice, too, I don’t know.

**Comment by dfurber** [comment]
No, it won't be clickable, because it would go to the same place as the name.

**Comment by PatientRowan** [comment]
It would go the same place as the name if the point of the click is to open the record.  But I also want to go to the point in the index where the name or name variant is filed, to look around.  It would be most helpful for common names where I get taken from Browne to Brown but want to see the other William Browns because I suspect that there has been a mixup (conflated identies on one record or one person split into two or more records); sometimes also to see family members with a similar name problem (today I worked on the Stark/Starke families)
But this is far from a requirement, I can always just adjust the search and scroll.
